CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-- 
+- [Issue #84](https://github.com/newrelic/newrelic-logenricher-dotnet/issues/84) is fixed.  This was a problem with how the log message level was being reported to New Relic from the Log4Net layout.
```

Comment:
How about something likes:

Fixes [Issue #84](https://github.com/newrelic/newrelic-logenricher-dotnet/issues/84) where the log4net log enricher incorrectly extracts log level value to report to New Relic.

Reply:
Sounds good, I updated it.